exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/DrillRoot.java

```diff
@@ -189,7 +189,7 @@ public ClusterInfo getClusterInfoJSON() {
     String adminUsers = ExecConstants.ADMIN_USERS_VALIDATOR.getAdminUsers(optionManager);
     String adminUserGroups = ExecConstants.ADMIN_USER_GROUPS_VALIDATOR.getAdminUserGroups(optionManager);
 
-    final boolean shouldShowUserInfo = isUserLoggedIn &&
+    final boolean shouldShowAdminInfo = isUserLoggedIn &&
```

Comment:
I think it is not fully correct to use this option to decide if we can show shutdown button or not.
This option is used only when authentication is enabled. When it's not, user by default is admin. So with your fix without authentication, this button is also absent.